src/controllers/tripsController.js

```diff
@@ -66,31 +61,26 @@ export default class requestController {
    * @return {object} request
    */
   static async createReturnRequest(req, res) {
-    const { user } = req;
-    const {
-      location,
-      destination,
-      departureDate,
-      returnDate,
-      reason,
-      accomodation,
-      gender, role, passportName
-    } = req.body;
-    if (Date.parse(departureDate) > Date.parse(returnDate)) {
-      return Response.errorResponse(res, 400, res.__('the return date must be greater than departure date'));
-    }
     try {
+      const { user } = req;
+      const {
+        location,
+        destination,
+        departureDate,
+        returnDate,
+        reason,
+        gender, role, passportName
+      } = req.body;
+      if (Date.parse(departureDate) > Date.parse(returnDate)) {
```

Comment:
If the dates are equal, it means. I can book an accommodation where I won't spend a night. Is this the intended result?

Reply:
We changed it to allowing a user to spend at least one night!!